### DISCUSSION TITLE
Where can I find the older network T30 (28M resnet)?

### DISCUSSION THREAD

**Discussion by jiacheng-ye** [LeelaChessZero/lc0#1980] [Q&A] [answered]

**Comment by mooskagh** [comment, marked as answer]
The T30 network mentioned in [this blog post](https://lczero.org/blog/2024/02/how-well-do-lc0-networks-compare-to-the-greatest-transformer-network-from-deepmind/) is 32930 which can be downloaded [here](https://training.lczero.org/get_network?sha=5c222ccd1ccbed2666b3a8ef94d8833d386d2168d51c3e99a0a3fc37a56d2569).

(the way to look up yourself would be to load [this page](https://training.lczero.org/networks/?show_all=1), but that page is huge and likely to crash a browser. We should probably do something about that).
